lib/assessment.rb

```diff
@@ -5,7 +5,8 @@ class Assessment
   # enumerable supplied.  Assume each entry implements a length
   # method that returns a Fixnum.
   def longest(enum)
-    # Your code goes here
+    enum.max_by { |x| x.length }
+    # I didn't figure out how to solve this, but was testing ways to use the above line found at http://ruby-doc.org/core-2.1.0/Enumerable.html#method-i-max_by
```

Comment:
Researching is almost as good as knowing; even though you didn't fully understand, great job digging!